README.md

```diff
@@ -11,7 +11,7 @@ an excellent place to start.
 To enter an environment without cloning this repository you can run:
 
 ```
-nix-shell https://github.com/alpmestan/ghc.nix/archive/master.tar.gz
+nix-shell https://github.com/alpmestan/ghc.nix/archive/master.tar.gz --attr devShells.<your-system>.default
```

Comment:
I think an explanation what `<your-system>` could be, would be good. Likely with an example.

Reply:
Thank you, I have fixed this.